app/javascript/components/explore/ExploreView.jsx

```diff
@@ -43,15 +43,6 @@ function RoutableExploreTab({ studyAccession }) {
 
   useEffect(() => {loadStudyData()}, [studyAccession])
 
-  useEffect(() => {
-    // if the user hasn't selected anything, and there are genelists to view, but no clusters
-    // default to the first gene list
-    if ((exploreInfo && exploreInfo.annotationList.clusters.length === 0 &&
```

Comment:
Do we need to be careful with undefined annotionLists or clusters here? aka there be '?' checks? aka `exploreInfo.annotationList?.clusters?.length` ?